templates/components/itilobject/timeline/approbation_form.html.twig

```diff
@@ -45,7 +47,7 @@
                <input type="hidden" name="_glpi_csrf_token" value="{{ csrf_token() }}" />
                <input type="hidden" name="itemtype" value="{{ item.getType() }}" />
                <input type="hidden" name="items_id" value="{{ item.fields['id'] }}" />
-               <input type="hidden" name="requesttypes_id" value="{{ item.fields['id'] }}" />
+               <input type="hidden" name="requesttypes_id" value="{{ item.fields['requesttypes_id'] }}" />
```

Comment:
I discussed about it with @flegastelois. We do not want to display the request type dropdown here, and we do not want either to reuse the value from the parent item. Using the `0` value is the best option here, it will just consider that the information is missing, and will result in not displaying the information.

<img width="898" height="293" alt="image" src="https://github.com/user-attachments/assets/c875d022-ebee-4514-863b-80ce3352bfde" />

```suggestion
               <input type="hidden" name="requesttypes_id" value="0" />
```